pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```
@@ -171,10 +173,19 @@ public HashMap<String, MercuryOrderDto> getPossibleBarcodesForParticipant(String
 
             return dbVals;
         });
+
+        if (results.resultException != null) {
+            throw new RuntimeException("Error getting the mercury orders", results.resultException);
+        }
         log.info(String.format("Found %d possible barcodes for participant %s", map.size(), ddpParticipantId));
         return map;
     }
 
+    @Override
+    public int create(MercuryOrderDto mercuryOrderDto) {
+        return 0;
```

Should this really throw something like ("Not implemented") instead of returning 0?